omni-relayer/src/workers/evm.rs

```diff
@@ -73,3 +73,65 @@ pub async fn finalize_withdraw(
         .await;
     }
 }
+
+pub async fn claim_native_fee(
+    redis_client: redis::Client,
+    connector: Arc<OmniConnector>,
+) -> Result<()> {
+    let redis_connection = redis_client.get_multiplexed_tokio_connection().await?;
+
+    loop {
+        let mut redis_connection_clone = redis_connection.clone();
+        let Some(events) = utils::redis::get_events(
+            &mut redis_connection_clone,
+            utils::redis::ETH_WITHDRAW_EVENTS.to_string(),
+        )
+        .await
+        else {
+            tokio::time::sleep(tokio::time::Duration::from_secs(
+                utils::redis::SLEEP_TIME_AFTER_EVENTS_PROCESS_SECS,
+            ))
+            .await;
+            continue;
+        };
+
+        let mut handlers = Vec::new();
+        for (key, event) in events {
+            if let Ok(event) = serde_json::from_str::<Nep141LockerEvent>(&event) {
+                handlers.push(tokio::spawn({
+                    let mut redis_connection = redis_connection.clone();
+                    let connector = connector.clone();
+
+                    async move {
+                        let Nep141LockerEvent::SignClaimNativeFeeEvent { .. } = event else {
+                            warn!("Expected SignClaimNativeFeeEvent, got: {:?}", event);
+                            return;
+                        };
+
+                        log::info!("Received SignClaimNativeFeeEvent log");
+
+                        match connector.evm_claim_native_fee_with_log(event).await {
+                            Ok(tx_hash) => {
+                                log::info!("Finalized withdraw: {:?}", tx_hash);
+                                utils::redis::remove_event(
+                                    &mut redis_connection,
+                                    utils::redis::ETH_WITHDRAW_EVENTS,
```

Comment:
ETH_WITHDRAW_EVENTS?

Reply:
Fixed in f218c04b15a5f0c5f72a4440b203d11449842b17